invenio/base/scripts/database.py

```diff
@@ -86,6 +86,9 @@ def init(user='root', password='', yes_i_know=False):
             (cmd_prefix + '-e "GRANT ALL PRIVILEGES ON '
              '{CFG_DATABASE_NAME}.* TO {CFG_DATABASE_USER}@localhost '
              'IDENTIFIED BY {CFG_DATABASE_PASS}"'),
+            (cmd_prefix + '-e "GRANT ALL PRIVILEGES ON '
+             '{CFG_DATABASE_NAME}.* TO {CFG_DATABASE_USER}@\'%\' '
+             'IDENTIFIED BY {CFG_DATABASE_PASS}"'),
```

Comment:
This is quite open. We should document that this is recommended only for development instances, and possibly print a warning...